test/playlist.test.js

```diff
@@ -686,6 +686,95 @@ QUnit.test('estimates segment request time based on bandwidth', function(assert)
   assert.equal(estimate, 8, 'takes into account bytes already received from download');
 });
 
+QUnit.module('Playlist enabled states', {
+  beforeEach(assert) {
+    this.env = useFakeEnvironment(assert);
+    this.clock = this.env.clock;
+  },
+  afterEach() {
+    this.env.restore();
+  }
+});
+
+QUnit.test('determines if a playlist is incompatible', function(assert) {
+  // incompatible means that the playlist was blacklisted due to incompatible
+  // configuration e.g. audio only stream when trying to playback audio and video.
+  // incompaatibility is denoted by a blacklist of Infinity.
+  const playlist = {};
```

Comment:
Similar to my comment below, you could just get rid of `const playlist` and pass the raw values into the method under test.